<launch>
  <node pkg="robot_self_filter" type="configurator.py" name="robot_self_filter_configurator" output="screen">
    <rosparam command="load" file="$(find robot_self_filter)/config/robot_self_filter.yaml"/>
  </node>

  <arg name="robot_filter_pc_src" default="/photoneo_center/pointcloud" />
  <arg name="robot_filter_pc_dst" default="/robot_filtered_cloud" />
  <node pkg="timed_roslaunch" type="timed_roslaunch.sh" name="timed_roslaunch" args="2 robot_self_filter self_filter.launch robot_filter_pc_src:=$(arg robot_filter_pc_src) robot_filter_pc_dst:=$(arg robot_filter_pc_dst)" output="screen" />

</launch>
